sync/handlers.py

```diff
@@ -280,6 +280,18 @@ def __call__(self, git_gecko, git_wpt, body):
 
         with SyncLock.for_process(sync.process_name) as lock:
             with sync.as_mut(lock), try_push.as_mut(lock):
+                if try_push.taskgroup_id is None:
```

Comment:
Can you add a comment here explaining that this is actually a thing that happens? I wonder if we should also record an error in new relic in this case or something so we can track it more easily.

Reply:
Will add a comment. I was thinking we could use [record_custom_event](https://docs.newrelic.com/docs/agents/python-agent/python-agent-api/record_custom_event) but it doesn't look like that goes to the same general area that the Errors go to, instead going to the Insights tab? not sure how useful that is

Reply:
Hmm, I haven't really used the insights tab much, so I guess it's OK to skip that for now if you aren't moved to try it.

Reply:
I will look into it, it looks like it could be useful throughout the codebase